keyboards/clickety_split/leeloo/info.json

```diff
@@ -1,13 +1,19 @@
 {
-  "keyboard_name": "Leeloo",
   "manufacturer": "Clickety Split Ltd",
   "url": "https://clicketysplit.ca/pages/leeloo",
   "maintainer": "ClicketySplit",
   "usb": {
     "vid": "0x4013",
-    "pid": "0x2022",
-    "device_version": "0.1.3"
+    "pid": "0x2022"
   },
+  "diode_direction": "COL2ROW",
+  "matrix_pins": {
+      "cols": ["F6", "F7", "B1", "B3", "B2", "B6"],
+      "rows": ["C6", "D7", "E6", "B4", "B5"]
+  },
+  "split": { "soft_serial_pin": "D2" },
+  "tapping": { "term": 150 },
```

Comment:
Generally not preferred to set this at the keyboard level, especially lowering it from the default.

Reply:
Thank you, @waffle87; I'll remove the tapping term.